### DISCUSSION TITLE
Implementing SQLite for storing logs

### DISCUSSION THREAD

**Discussion by BOSS294** [BOSS294/AutoMatic-File-Generator---PYTHON--#5] [Ideas]
### SQL LITE INTEGRATION 

Let's integrate SQLITE with our software so that it can record the total logs

Which directory was created by who , When & Where

GUI will now have buttons to show Creating Logs

**Comment by jindalpriyanshu101** [comment]
I guess it won't be good to store user data into SqLite, as such things are entitled to data privacy. But we can surely store the data using sqlite into a .db file, But that'll be created locally only accessible to user.

Will contain logs of last execution only, so it'll be easy to debug about what folders were created last time?

**Reply by BOSS294** [reply]
Using sqlite because our software is for local use only so I thought it would be best to storedata on local disks using sqlite
